exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/statements/api/ExposedDatabaseMetadata.kt

```diff
@@ -67,6 +67,9 @@ abstract class ExposedDatabaseMetadata(val database: String) {
     /** Returns a map with the [PrimaryKeyMetadata] in each of the specified [tables]. */
     abstract fun existingPrimaryKeys(vararg tables: Table): Map<Table, PrimaryKeyMetadata?>
 
+    /** Returns a list of the names of all sequences in the database. */
+    abstract fun existingSequences(): List<String>
```

Comment:
It looks like `existing` is an obsolete word in these kinds of functions

Reply:
Not sure I got what you mean. Do you mean in Exposed? We're using it in several other functions. Could you please clarify?